frontend/src/core/observations/ObservationRemoveAssessment.tsx

```diff
@@ -84,7 +84,7 @@ const ObservationRemoveAssessment = () => {
                 <DialogTitle>Observation Remove Assessment</DialogTitle>
                 <DialogContent>
                     <SimpleForm onSubmit={observationUpdate} toolbar={<CustomToolbar />}>
-                        <TextInputWide source="comment" validate={validate_required_255} />
+                        <TextInputWide source="comment" validate={validate_required} />
```

Comment:
```suggestion
                        <TextInputWide source="comment" validate={validate_required} minRows={3} />
```

and see remark about the size limitiation in the model